### DISCUSSION TITLE
population pyramid

### DISCUSSION THREAD

**Discussion by ghost** [Abhinandan-Kushwaha/react-native-gifted-charts#480] [General]
Anyone help me to develop a population pyramid.... based on male and female, like below example
![image](https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/assets/47064533/ee908c89-6f36-4553-9c30-2444a4327a4f)

**Comment by ak-spotter** [comment]
Hi @nehalasim 
Pyramid charts are now supported from version `1.3.29` onwards🎉

See https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/issues/481#issuecomment-1877943263

**Comment by ghost** [comment]
Thank you very much for the new update....it will be very helpful
